internal/game/room.go

```diff
@@ -0,0 +1,104 @@
+package game
+
+import (
+	"errors"
+
+	"github.com/gorilla/websocket"
+	"github.com/oklog/ulid/v2"
+	"go.uber.org/zap"
+)
+
+var (
+	ErrRoomFull = errors.New("room is full")
+)
+
+// One game instance takes place in each room.
+// The UID can be used to invite other players to the room.
+// There must be at least two people in a room for the game to start.
+// The maximum is five per room. One player is represented by a connection.
+type room struct {
```

Comment:
It's been a minute since I last did stuff with Go. Can structs be private?

Reply:
yes